package.json

```diff
@@ -51,22 +54,30 @@
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-scanjs-rules": "^0.2.1",
     "eslint-plugin-security": "^1.5.0",
-    "eslint-plugin-simple-import-sort": "^8.0.0",
     "eslint-plugin-sonarjs": "^0.16.0",
     "eslint-plugin-switch-case": "^1.1.2",
     "eslint-plugin-unicorn": "^45.0.0",
     "fs-extra": "^8.1.0",
+    "postcss-import": "^15.1.0",
+    "postcss-nesting": "^12.0.0",
     "prettier": "^2.8.1",
     "prettier-eslint": "^15.0.1",
     "prettier-eslint-cli": "^7.1.0",
-    "rollup": "^2.77.3",
+    "rollup": "^3.0.0",
+    "rollup-plugin-import-css": "^3.2.1",
+    "rollup-plugin-inline-svg": "^3.0.3",
     "rollup-plugin-livereload": "^2.0.5",
+    "rollup-plugin-postcss-modules": "^2.1.1",
     "rollup-plugin-scss": "^3.0.0",
     "rollup-plugin-stylelint": "^1.0.0",
-    "rollup-plugin-terser": "^7.0.2",
     "sass": "^1.25.0",
     "tslib": "^2.4.0",
     "typescript": "^4.7.4",
     "yargs": "^17.3.1"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.4.0",
+    "@fortawesome/fontawesome-svg-core": "^6.4.0",
+    "@fortawesome/free-solid-svg-icons": "^6.4.0"

```

Comment:
I forget what our conversation on this was. Why are these included as dependencies when Foundry includes fontawesome?